news_and_announcements.Rmd

```diff
@@ -0,0 +1,25 @@
+# News and Announcements
+
+Here are some resources that can be useful to keep up with the developments in R:
+
+## Blogs
+
+The [R project](https://www.r-project.org/) maintains a list of public [blogs](https://developer.r-project.org/Blog/public/) mainly written by the R Core Development Team. Daily news about R-devel can be found [here](https://developer.r-project.org/blosxom.cgi/R-devel).
+
+## Conferences
+
+Updates about conferences actively supported or endorsed by The R Foundation can be found [here](https://www.r-project.org/conferences/). These conferences are organised by members from the R community.
+
+## Journal
+
+[The R Journal](https://journal.r-project.org/) is an open access and refereed journal featuring short to medium length articles that should be of interest to users or developers of R. It also has a news section where information on, changes in R (new features of the latest release), changes on CRAN (new add-on packages, manuals, binary contributions, mirrors, etc.), upcoming conferences, and conference reports is provided.
+
+
+## Mailing lists
+
+  * [R-announce](https://stat.ethz.ch/mailman/listinfo/r-announce): A moderated mailing list used for announcements by the R Core Development Team. Major announcements about the development of R and the availability of new code are made here.
+  * [R-packages](https://stat.ethz.ch/mailman/listinfo/r-packages): A moderated mailing list for announcements about contributed R packages (typically on [CRAN](https://cran.r-project.org/)) and similar R project extensions.
+
+## Twitter
+
+Follow ['@R_dev_news'](https://twitter.com/R_dev_news) on Twitter for the [R development updates](https://developer.r-project.org/RSSfeeds.html) and [blog post](https://developer.r-project.org/Blog/public/) announcements.

```

Comment:
Thanks for mentioning the account R_dev_news, but it is not official and don't think it should be mentioned here. Instead, as it was pointed out on the mailing list, there should be a mention to the @_R_foundation account, which also provides information about grants, the blog and other relevant content. 
Perhaps there should mention that exists some other accounts which provide the same information from these and other source (I can think about the cranberries about updates on CRAN packages, the account that tracks CRAN policy changes..., R_dev_news). 

The section name could be changed to social media with a section for other social site where news and updates are shared (linkedin, facebook, xing, instagram, tick-tock?).

Reply:
Perhaps stick with the official resources and mention at the end that there are other unofficial resources too numerous to list here.